madgiraffestudios/src/App.js

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import './App.scss';
```

Comment:
There's a routes.js, index.js, App.js -- consider what should go into what file...I don't think routes belong in App.js except for components itself